### ISSUE TITLE
Generation of documentation

### ISSUE THREAD

**Issue by Eiyeron** [Siapran/Generic-Container-Lib#1]
Hi.

I'm digging into your repository to check how ti's made and here's something that could be useful : generate a documentation.

Usually, doc blocks are made the same way than Javas `/** comment */`, and tools like [Doxygen]http://www.stack.nl/~dimitri/doxygen/) can be set in a workflow to generate multiple sorts of documentations like HTML or PDF.

**Comment by siapran** [comment]
Sure. I was planning on doing that after I finished implementing a few other containers.

**Comment by siapran** [comment]
also i'm not sure doxygen will work correctly since the function names are generated by macros

**Comment by Eiyeron** [comment]
RIght-o, so It gonna be harder to use such a package manager. It could
probably try simething while fetching `/** doc comments */` but that's
something to test locally.

2015-04-01 17:46 GMT+02:00 Siapran notifications@github.com:
